test/unittests/SettingsTest.php

```diff
@@ -151,6 +151,22 @@ public function testIsSandboxReturnsFalse($sandboxVal)
         $this->assertFalse($this->_settings->isSandbox());
     }
 
+    /**
+     * Test that getBaseURL() calls getSetting('www') and returns the correct string
+     *
+     * @return void
+     * @covers Settings::getBaseURL
+     */
+    public function testGetBaseURL()
```

Comment:
Seems to be missing the code path of the LORIS_BASEURL environment variable being set

Reply:
Hey @driusan , let me know if my latest commit is good!